css-pseudo-4/Overview.bs

```diff
@@ -1140,23 +1140,35 @@ Security and Privacy Considerations</h3>
   the styling of such highlighted segments.
 
 <h2 id="treelike">
-Tree-Abiding Pseudo-elements</h2>
+Tree-Abiding and Part-Like Pseudo-elements</h2>
 
   <dfn export lt="tree-abiding|tree-abiding pseudo-element">Tree-abiding pseudo-elements</dfn> always fit within the box tree.
   They <a lt="inheritance">inherit</a> any inheritable properties from their <a>originating element</a>;
   non-inheritable properties take their <a>initial values</a> as usual.
   [[CSS-CASCADE-4]]
 
-<h3 id=partlike>
-Part-Like Pseudo-Elements</h3>
-
   A subset of [=tree-abiding pseudo-elements=],
   the <dfn>part-like pseudo-elements</dfn>,
   have slightly stronger requirements:
   they act as if they have a well-defined location in the document tree.
   This enables them to interact with some other platform features
   as if they were real elements.
 
+  <div class=example>
+    For example, a [=part-like pseudo-element=]
+    can be used in the <{html-global/exportparts}> attribute,
+    to masquerade as a ''::part()''
+    for the component it's in:
+
+    <xmp class=html>
+      <template id=custom-element-template>
+        <p exportparts="::before : preceding-text">
+          You can style my ::before pseudo-element
+          by using ::part(preceding-text), too!
+      </template>
+    </xmp>
+  </div>
+
```

Comment:
This isn't helping to illustrate pseudo-elements that "interact with some other platform features as if they were real elements". It's illustrating a real element acting like a pseudo-element (by being selected by one), which is kindof the opposite!

Reply:
I think you're misunderstanding the example, because it illustrates the exact opposite. `::part(preceding-text)` is selecting a pseudo-element (the `p::before` pseudo).